test/core_tests.conf

```diff
@@ -42,13 +44,21 @@ module.exports = function(config) {
     {pattern: 'genfiles/core/js/cwc_blocks.js'},
     {pattern: 'genfiles/core/js/cwc_ui.js', watched: false},
 
-    {pattern: 'genfiles/core/css/**/*', included: false},
+    {pattern: 'genfiles/core/css/**/*', included: true},
```

Comment:
The css files are loaded by the test html and not included by default to detect possible issues.